keyboards/keychron/k12/rgb/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2022 Dimitris Mantzouranis <d3xter93@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define DIP_SWITCH_PINS {B6, D4}
```

Comment:
this can be data driven now, see https://github.com/SonixQMK/qmk_firmware/commit/98530cad3ba8733d8a100e0bc5f3cf47339c4b3e